### ISSUE TITLE
Polyline with drop segments preview is skewed instead of flat.

### ISSUE THREAD

**Issue by stevegiron** [xLightsSequencer/xLights#3625]
Instead of using multiple icicle segments to match my inverted V roofline, I used the polyline with drops. 
Sequencing works fine, output works fine.

The apparent bug is when viewing a multi-segment polyline (issue accentuated with drops), the model is not square to the screen but artificially rotated upon the Z axis.

To Reproduce
Create a new polyline model that looks like this   ^--^------^---  (that has 9 segments)
Set pixel count to 360
Set the drops to 3,4,5
Add the model to a sequence
Put a bars effect on the model and select the bars effect with model preview on.
The model does not appear flat on the screen as drawn. The right edge of the model has been pulled into the Z axis by 30-40 degrees.

The model looks ok in the house preview window.

![image](https://user-images.githubusercontent.com/10568498/206096651-69cd0f60-ad86-4c4a-87dd-27a4cab3e3f7.png)

xLights 22.23 Windows

**Comment by AzGilrock** [comment]
<img width="1290" alt="Screenshot 2022-12-07 at 5 02 37 PM" src="https://user-images.githubusercontent.com/2024669/206323511-b0426de0-cd5b-4f68-b284-601214051307.png">

**Comment by AzGilrock** [comment]
Yeah this checkin sounds like it was fixing this exact issue.  Maybe I un-fixed it when I worked on it...lol.  What's hard is Keith does so many formatting changes I'm having trouble finding the real code changes in that checkin.

**Comment by AzGilrock** [comment]
Well from what I can tell all the code Keith added for that fix is still there.  Not sure why it doesn't fix it anymore.

**Comment by stevegiron** [comment]
Well, render and playback is ok. Just looks silly on the model preview.

**Comment by AzGilrock** [comment]
Well what is happening is the Model Preview window tries to stretch the model to fill the full window height.  So if there is any deviation in Y positions its going to create a scaling matrix that stretches the top Y value up to the top of the window.  Try manually entering position values for all the segments that are on the lower vertical and type in the exact same value and then see how it does.  That fixed it for me on my test PolyLine where I drew eaves like yours.

**Comment by AzGilrock** [comment]
Hmm...that doesn't work either once you add the drops.  Really not sure how to improve it at this point.

**Comment by stevegiron** [comment]
How about adding multi-segment capability to the existing icicle effect since the preview works.

**Comment by cybercop23** [comment]
Is this still needed? If not, please close.

**Comment by AzGilrock** [comment]
Well it would have been nice to fix it but I didn't find a solution when I tried and I created the PolyLine model
